### PR TITLE
[No QA] Update copy for deleting a workspace

### PR DESCRIPTION
**Pull request by thienlnam**
### Fixed Issues

$ https://github.com/Expensify/App/issues/6722

### Tests

1. Create a workspace
2. Make sure the request to delete the workspace fails by disabling wifi or something similar
3. Make sure you see the failure growl

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### CODE REVIEW

File: src/languages/es.js [hunk 1]
```diff
@@ -680,6 +680,7 @@ export default {
             growlMessageOnSave: '¡La configuración del espacio de trabajo se ha guardado correctamente!',
             growlMessageOnDelete: 'Espacio de trabajo eliminado',
             deleteConfirmation: '¿Estás seguro de que quieres eliminar este espacio de trabajo?',
+            growlMessageOnDeleteError: 'No se puede eliminar el espacio de trabajo porque tiene informes que están siendo procesados.',
```
Comment (luacmartins) [inline src/languages/es.js:683]: I noticed that the english version does not have a period in the end. I think we should remove it here.

**Review by luacmartins — CHANGES_REQUESTED** [review]

File: src/languages/es.js (hunk 1 above)
Comment (jasperhuangg) [reply src/languages/es.js:683]: +1!

**Review by jasperhuangg — DISMISSED** [review]
Looks good minus the comment from @luacmartins

**Comment by thienlnam** [comment]
periodt removed

**Review by luacmartins — APPROVED** [review]
LGTM!

**Pull request merged**